### ISSUE TITLE
feature request: exponents for long strings of ups and downs

### ISSUE THREAD

**Issue by TallKite** [m-yac/xen-calc#17]
if we input 65\205, get vvvvvM3, should get v⁵M3

related: 67\205 is called 3-down major 3rd, IMO it should be called triple-down. I'm on the fence about quadruple & quintuple vs. 4-down & 5-down. I'm with you on using 6-down instead of sextuple-down.

**Comment by m-yac** [comment]
Thanks for the suggestion! My concern with this is that it doesn't have an ASCII equivalent. Specifically, how would you type "v⁵M3" without using a unicode character? Certainly not "v^5M3" like the way all other exponents are written, since a 5-up major third would confusingly be written "^^3M3".

Perhaps a solution would be to use the notation "v5M3"? This would of course be rendered as "v⁵M3" in the output, but "v5M3" would be how it would need to be typed as input, and would be what appears in the URL. What are your thoughts?

**Comment by TallKite** [comment]
I was only thinking about the output. The output could certainly use unicode, e.g. v⁵M3, right?

For the input, I agree ^^3M3 is too confusing. Just like you have buttons for bb, b, etc. there could be an exponentiate button. Is it possible that you highlight the number, then click the exponentiate button to turn it into the unicode character? Or if the webapp doesn't know what's highlighted, maybe it works on the last number in the input line. So you type ^3, hit exponentiate, get ^³, the continue typing M3.

Another idea, the user could use parentheses somehow? ^(^3)M3 or (^)^3M3 or ((^)^3)M3?

**Comment by TallKite** [comment]
Another idea, spell out one of the two "^" symbols, so up^3 or ^exp3?
For any of these ideas, there would probably be a paragraph on the about page letting the user know it's a possibility.

**Comment by TallKite** [comment]
You mentioned "v5M3", which renders as "v⁵M3". This would work too, and it's simpler than my ideas.

**Comment by m-yac** [comment]
This has been added - for the ASCII version I ended up going with "(v)^6M3" if the exponent is a single digit and "(v)^(10)M3" otherwise. This is to avoid conflict with the notation "2A4" for multi-augmented/diminished intervals (e.g. in the old notation, does "^2A4" mean "double-up aug 4th" or "up 2-aug 4th"?).

For now I added "triple", as per your other issue, but not any other "-tuple" words.

Let me know if you find any problems!
